### PR TITLE
CASMHMS-6189 - Update for fixes to Rapid Restore procedures for release/1.5

### PR DESCRIPTION
**Pull request by mbuchmann-hpe**
# Description

Minor updates to docs and scripts for the Rapid Restore Procedures
Moved HSM scripts to correct location

CASMHMS-6189

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

### CODE REVIEW

**Comment by mbuchmann-hpe** [comment]
Ignoring failing script checks - Scripts are good.

File: scripts/operations/hardware_state_manager/backup_smd_postgres.sh
Comment (jwlv) [inline scripts/operations/hardware_state_manager/backup_smd_postgres.sh:1]: license-check is failing for this file due to not having a copyright year.

Comment (jwlv) [inline scripts/operations/hardware_state_manager/backup_smd_postgres.sh:1]: Additionally, the mvdan/shfmt check is failing in some way related to this file but looking at the output I have no idea why...

**Review by jwlv — CHANGES_REQUESTED** [review]

File: scripts/operations/hardware_state_manager/backup_smd_postgres.sh
Comment (mbuchmann-hpe) [reply scripts/operations/hardware_state_manager/backup_smd_postgres.sh:1]: Updated

Comment (mbuchmann-hpe) [reply scripts/operations/hardware_state_manager/backup_smd_postgres.sh:1]: It does not like the indenting in the script.

**Review by jwlv — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**